doc-content/enterprise-only/project-data/select-branches-proc.adoc

```diff
@@ -0,0 +1,12 @@
+[id='select-branches-proc']
+
+= Selecting branches
+You can switch between branches to test them and review their assets.
```

Comment:
More importantly you can switch between branches to develop the assets :) And development can go on in each branch separately, or something

Reply:
Modified to clarify.